controller/controller_manager.go

```diff
@@ -81,7 +81,7 @@ func StartControllers(stopCh chan struct{}, controllerID, serviceAccount, manage
 	priorityClassInformer := kubeInformerFactory.Scheduling().V1().PriorityClasses()
 
 	// TODO: [#1724] improve the logging setup/infrastructure further down the line
-	logger := logrus.StandardLogger()
+	logger := logrus.StandardLogger().WithField("node", controllerID)
```

Comment:
I am debating if we should include it here since every manager is running on different nodes so the information is embedded in the manager's name. But on the other side, searching will be easier with the nodeID.

Reply:
having the dedicated node field for each log output, seems like a good idea?
I don't see no harm in this and it will make it easier to correlate logs from different components.